service/src/main/java/uk/gov/hmcts/reform/fpl/service/LegalRepresentativeService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentativesChange;
+
+import java.util.List;
+
+import static java.util.Collections.emptySet;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LegalRepresentativeService {
+
+    private final CaseService caseService;
+    private final OrganisationService organisationService;
+    private final LegalRepresentativesDifferenceCalculator legalRepresentativesDifferenceCalculator;
+
+    public void updateRepresentatives(Long caseId,
+                                      List<LegalRepresentative> original,
+                                      List<LegalRepresentative> updated) {
+
+        LegalRepresentativesChange legalRepresentativesChange = legalRepresentativesDifferenceCalculator.calculate(
```

Comment:
very minor: `legalRepresentativesDifferenceCalculator` to `differenceCalculator`? Then from the service it can be assumed that it is about legal reps and then you can fit method call on one line 🙂

Reply:
done